azurerm/internal/services/storage/storage_account_resource.go

```diff
@@ -239,6 +242,12 @@ func resourceStorageAccount() *pluginsdk.Resource {
 				Default:  false,
 			},
 
+			"allow_shared_key_access": {
```

Comment:
lets be consistent with other booleans
```suggestion
			"shared_key_access_enabled": {
```